bindata/v3.11.0/kube-apiserver/recovery-pod.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: kube-system  # for disaster recovery we need namespace that is always present
+  name: kube-apiserver-recovery
+  labels:
+    revision: "recovery"
+spec:
+  containers:
+  - name: kube-apiserver-recovery
+    image: "{{ .KubeApiserverImage }}"
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: ["hypershift", "openshift-kube-apiserver"]
```

Comment:
@sttts I think it may be simpler to launch `hyperkube kube-apiserver` and avoid any entanglments.  preference?

Reply:
Do we use it anywhere else? I'd not want to be the only one exploring that path and running it differently then the regular kube-apiserver.

Reply:
have no strong opinion either. If this works, let's leave it.